example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -144,8 +156,8 @@ describe('The S3 Ingest Granules workflow', () => {
     // clean up stack state added by test
     await Promise.all([
       deleteFolder(config.bucket, testDataFolder),
-      cleanupCollections(config.stackName, config.bucket, collectionsDir, testSuffix),
-      cleanupProviders(config.stackName, config.bucket, providersDir, testSuffix),
+      collectionModel.delete(collection),
+      providerModel.delete(provider),
```

Comment:
Will this fail if there is nothing there to delete? When Are we checking if something was actually created?